wdl/pipelines/TechAgnostic/Visualization/SvQCPlots.wdl

```diff
@@ -0,0 +1,451 @@
+    }
+
+    Int minimal_disk_size = (ceil(size(all_stats, "GB"))  + 100 ) # 100GB buffer
+    Int disk_size = if minimal_disk_size > 100 then minimal_disk_size else 100
+
+    command<<<
+        set -euo pipefail
+
+        for caller in ~{sep=" " callers}
+        do
+            for stat_file in ~{sep=" " all_stats}
+            do
+                if [[ ${stat_file} =~ ${caller} ]]
+                then
+                    cat ${stat_file} >> ${caller}_all_SV_lengths_by_type.txt
+                fi
+            done
+        done
+
+        # Check if the files created match the number of callers
+        if [[ $(find ./*_all_SV_lengths_by_type.txt | wc -l) -ne ~{length(callers)} ]]
+        then
+            echo "ERROR: Number of callers does not match number of files created"
+            ls *_all_SV_lengths_by_type.txt
+            exit 1
+        fi
+
+    >>>
+
+    output{
+        Array[File] all_stats_by_type = glob("*_all_SV_lengths_by_type.txt")
+    }
+
+    #########################
+    RuntimeAttr default_attr = object {
+        cpu_cores:          2,
+        mem_gb:             8,
+        disk_gb:            disk_size,
+        boot_disk_gb:       10,
+        preemptible_tries:  3,
+        max_retries:        0,
+        docker:             "us.gcr.io/broad-dsp-lrma/lr-basic:latest"
+    }
+    RuntimeAttr runtime_attr = select_first([runtime_attr_override, default_attr])
+    runtime {
+        cpu:                    select_first([runtime_attr.cpu_cores,         default_attr.cpu_cores])
+        memory:                 select_first([runtime_attr.mem_gb,            default_attr.mem_gb]) + " GiB"
+        disks: "local-disk " +  select_first([runtime_attr.disk_gb,           default_attr.disk_gb]) + " HDD"
+        bootDiskSizeGb:         select_first([runtime_attr.boot_disk_gb,      default_attr.boot_disk_gb])
+        preemptible:            select_first([runtime_attr.preemptible_tries, default_attr.preemptible_tries])
+        maxRetries:             select_first([runtime_attr.max_retries,       default_attr.max_retries])
+        docker:                 select_first([runtime_attr.docker,            default_attr.docker])
+    }
+}
+
+task compileSVstats {
+
+    parameter_meta{
+        samples: "List of sample names"
+        all_stats: "List of files containing SV stats for each sample"
+        callers: "List of SV callers used to generate input vcf files"
+    }
+
+    input {
+        Array[String] samples
+        Array[File] all_stats
+        Array[String] callers
+        RuntimeAttr? runtime_attr_override
+    }
+
+    File sampleFile = write_lines(samples)
+
+    Int minimal_disk_size = (ceil(size(all_stats, "GB") ) + 100 ) # 100GB buffer
+    Int disk_size = if minimal_disk_size > 100 then minimal_disk_size else 100
+
+
+    command <<<
+        set -euo pipefail
+
+        mkdir -p stats_by_sample
+        for file in ~{sep=" " all_stats}
+        do
+            mv $file ./stats_by_sample
+        done
```

Comment:
given that the Python code below is long enough (60 lines), it's worth making a docker. 
That docker can base-off of the minimal python image and just `COPY` a script that has the following lines as its content.

Reply:
done